content/contributor/social-service/2-login.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Login
+url: /contributor/social-service/login
+weight: 2
+---
+
+# Login to Social Service
+
+## Description
+
+The authentication process for the Social Service involves interacting with the Matrix server. Users must log in to Matrix using their address, signing a fetch operation to obtain an authentication token. This token is then used to authorize interactions with the Social Service.
+
+## HTTP Request
+
+To acquire the authentication token, a POST request must be made to the following URL: `https://social.decentraland.org/_matrix/client/r0/login`
```

Comment:
We can add here that for this we need to create and sign a message with AuthChain and link to the docs section.

Also comment that a more complete example will be shown below to understand how it's used.